Module4.3_Search_and_Sorting_Algorithms/activities/Act1_SearchingAndSorting/2.md

```diff
@@ -102,45 +99,46 @@ Before we get into the meat of the algorithm, skim this diagram. As you read the
 
 ![](https://i.imgur.com/cyxuppV.jpg)
```

Comment:
This graphic is really amazing!! But displayed inside the frame, it looks really compressed. I think this overall graphic should still be provided, however the graphic should be split up into each step, and each step should be associated with the appropriate chunk of text in the card